### PR TITLE
fix: repair refile to headline in the same file (#633)

### PR DESCRIPTION
**Pull request by seflue**
- use correct line position from target headline
- reset cursor back
- fix issue #633

### CODE REVIEW

**Comment by seflue** [comment]
@kristijanhusak, @jgollenz: I need your aproval for this tiny change.

File: lua/orgmode/capture/init.lua [hunk 1]
```diff
@@ -349,7 +349,10 @@ function Capture:_refile_to(opts)
 
   local item = opts.item
   if is_same_file and item then
-    vim.cmd(string.format('silent! %d,%d move %s', item.range.start_line, item.range.end_line, tostring(opts.file)))
+    local target = opts.range.end_line
+    local pos = vim.fn.getcurpos()
```
Comment (kristijanhusak) [inline lua/orgmode/capture/init.lua:353]: You can use `winsaveview()` and `winrestview()` to keep the cursor position.

**Review by kristijanhusak — CHANGES_REQUESTED** [review]

**Review by kristijanhusak — APPROVED** [review]

**Comment by kristijanhusak** [comment]
@seflue code looks good, but tests for nightly started to fail. Can you take a look?

**Comment by seflue** [comment]
I'm already having a look into it and I don't get it. The failing test is regarding a capturing (failed expectation) which should not trigger the branch.

**Comment by seflue** [comment]
I reverted the change to see, if the test failure was caused by it - unsuprisingly it wasn't. So the nightly version introduced a regression with the plugin, which is independent from this PR and needs to be investigated independently.

**Comment by kristijanhusak** [comment]
Ok, thanks for checking. I'm merging this in and we can investigate the issue separately.

**Pull request merged**